.github/workflows/ext_run_test_integration.yml

```diff
@@ -0,0 +1,33 @@
+name: external_run_test-integration
```

Comment:
Propose to name the workflow `test_clickhouse_version`.

Reply:
Changed

Reply:
Let's also change workflow file name for consistency.

Reply:
Done